pyproject.toml

```diff
@@ -113,10 +113,6 @@ docs = [
     "qcodes_loop>=0.1.1", # legacy dataset import examples
     "jinja2>=3.1.3", # transitive dependency pin due to cve in earlier version
 ]
-opencensus = [
```

Comment:
add newsfragment?

Reply:
@astafan8 Rather remember to push the fragment that I had already written ;)